src/plugin.js

```diff
@@ -5,17 +5,29 @@ import { join } from 'path';
 import fetch from 'node-fetch';
 import fs from 'fs';
 
-async function readEikJSONMaps(eikJSONPath) {
+async function readJSONFile(path) {
     try {
-        const contents = await fs.promises.readFile(eikJSONPath);
-        const eikJSON = JSON.parse(contents);
-        if (typeof eikJSON['import-map'] === 'string') return [eikJSON['import-map']];
-        return eikJSON['import-map'] || [];
+        const contents = await fs.promises.readFile(path);
+        return JSON.parse(contents);
     } catch (err) {
-        return [];
+        return {};
     }
 }
 
+async function readEikJSONMaps(eikJSONPath, pkgJSONPath) {
+    const eikJSON = await readJSONFile(eikJSONPath);
+    const pkgJSON = await readJSONFile(pkgJSONPath);
```

Comment:
Could this be refactored into running in parallel by turning `readJSONFile()` into returning a promise and then run these with `Promise.all()`?